ballerina/Package.md

```diff
@@ -19,6 +19,11 @@ import ballerinax/mssql;
 import ballerinax/mssql.driver as _;
 ```
 
+## Report issues
+
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina standard library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+
 # Useful Links
 * Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
```

Comment:
Change to discord

Reply:
Fixed in [3a3bc95](https://github.com/ballerina-platform/module-ballerinax-mssql.driver/pull/12/commits/3a3bc95c2762137089607341f212e75e7d544345)